packit_service/service/events.py

```diff
@@ -485,7 +485,7 @@ def __init__(
         self.target_repo_namespace = target_repo_namespace
         self.target_repo_name = target_repo_name
         self.https_url = https_url
-        self.username = username
+        self.user_login = username
```

Comment:
Why this change? (I just want to know.)

Reply:
To reuse `GitHubPullRequestCommentCoprBuildHandler` as it should have `self.event.user_login`. So just renaming username -> user_login.

Reply:
Thanks!